### ISSUE TITLE
Is Vth_opa a parameter optimized in isolation?

### ISSUE THREAD

**Issue by tansor** [zju-bmi-lab/SpikingGS#4]
Hello, thank you for your excellent work!
I find that ```gaussians.Vth_opa``` is initilized by 0.005, and optimized by loss ```ld_opa * 1 / (gaussians.Vth_opa + 1e-6)``` with a learning rate ```0.0002 -> 0.00014 -> 0.00005```.
I wonder if this can result a persistent growth (without end condition) of Vth_opa ?
Once if that is true, why Vth_opa is optimized in isolation, but not by other condition of 3dgs system?

**Comment by shippoT** [comment]
Hi, thank you for your interest in our work! Vth_opa will not lead to persistent growth, as its gradients will be counterbalanced by other losses. In fact, during training, the updates to Vth_opa are influenced by other losses due to the FIF-neuron design.
